templates/CRM/Admin/Form/Setting/UF.tpl

```diff
@@ -13,12 +13,14 @@
 <div class="crm-block crm-form-block crm-uf-form-block">
 <div class="crm-submit-buttons">{include file="CRM/common/formButtons.tpl" location="top"}</div>
       <table class="form-layout-compressed">
-         <tr class="crm-uf-form-block-userFrameworkUsersTableName">
+        {if !empty($form.userFrameworkUsersTableName)}
+        <tr class="crm-uf-form-block-userFrameworkUsersTableName">
             <td class="label">{$form.userFrameworkUsersTableName.label}</td>
             <td>{$form.userFrameworkUsersTableName.html}</td>
         </tr>
+        {/if}
         {if !empty($form.wpBasePage)}
```

Comment:
Our current trend is to get rid of any `empty()` or `isset()` from tpl files because they don't work with escaping, and just ensure the variable is always assigned.

Reply:
Ah, I knew that `isset` was problematic due to the escaping work @eileenmcnaughton has been doing, but I didn't realise `empty` was too?

Reply:
Right, we're avoiding both now

Reply:
Ok, updated. I sort of feel like there may be a smarter solution that what I've found - it would be nice if items in `_settings` could be set to `FALSE`, and then this be read from the template. But I wasn't seeing how, so I've introduced new `wpBasePageEnabled` and `userFrameworkUsersTableNameEnabled` smarty variables.

Reply:
@braders yep empy is a problem too because when you turn on default escaping it passes the value to the escape function (which gives a notice because you are using an undefined variable) before it goes to the empty function.

To enable - you need this in civicrm.settings.php 
```
if (!defined('CIVICRM_SMARTY_DEFAULT_ESCAPE')) {
  define('CIVICRM_SMARTY_DEFAULT_ESCAPE', TRUE);
}
```